app/javascript/Components/Greeting.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from "react";
+import { ReactDOM } from "react-dom";
+import { getMessage } from "../redux/Reducer";
+import axios from "axios";
+import { useDispatch, useSelector } from "react-redux";
```

Comment:
- [ ] Kindly ensure your packages are installed before trying to import them. There is no react-redux in your `package.json` file

  ![yhf](https://user-images.githubusercontent.com/54848363/197773390-3873049b-57ad-4be9-ad71-667d1408f5ba.jpg)